sys/fs/fuse/fuse_vnops.c

```diff
@@ -3226,7 +3232,7 @@ fuse_vnop_vptofh(struct vop_vptofh_args *ap)
                  */
 		SDT_PROBE2(fusefs, , vnops, trace, 1,
 			"VOP_VPTOFH with FUSE_OPENDIR");
-		return EOPNOTSUPP;
+		return (EXTERROR(EOPNOTSUPP, "FUSE_OPENDIR prevents VPTOFH"));
```

Comment:
Again, we shouldn't use VFS jargon.  How about "Server implements FUSE_OPENDIR so is not compatible with getfh."?